include/effectmanager.h

```diff
@@ -96,6 +96,7 @@ class EffectManager : public IJSONSerializable
             _clearTempEffectWhenExpired = true;
 
             // This is a hacky way to ensure that we start the correct effect after the temporary one
+            // REVIEW: Explain, I don't get it!

```

Comment:
The switching to the next effect is taken care of by NextEffect(), which starts with increasing _iCurrentEffect. We therefore need to decrease it here, to make sure that the first effect after the temporary one that is shown at startup - i.e. the NightDriver logo - is the one we want (either the then current one when the chip was powered off, or the one at index 0).

I've pushed a change to my issue-349 branch that adds this comment at this place in the code.